manuscript/240_Refactoring_Object_Composition.md

```diff
@@ -553,7 +553,7 @@ My main answer is: to deal with with complexity more effectively.
 
 What't complexity? For our purpose we can approximate it as a number of different decisions our application needs to make. As we add new features and fix errors or implement missed requirements, the complexity of our software grows. What can we do when it grows so larger than we are able to manage? We have the following choices:
 
- 1. Remove some decisions -- i.e. remove features from our application. This is very cool when we actually *can* do this, but there are times when this might be unacceptable from the business perspective.
+ 1. Remove some decisions -- i.e. remove features from our application. This is very cool when we *can* do this, but there are times when this might be unacceptable from the business perspective.
```

Comment:
:+1: (by the way, sorry for all those thumbs, my mind probably needs some marks of progress ;-)